src/util/runCommands.ts

```diff
@@ -0,0 +1,24 @@
+import { exec } from 'child_process';
+import { promisify } from 'util';
+
+const asyncExec = promisify(exec);
+
+export const runSequence = async(cmds: string[]) => {
+    try{
+        for await (const cmd of cmds) {
```

Comment:
Does this need to be a for await loop? You already await the asyncExec within the loop?

Reply:
taken this inspiration from src/util/index.ts and src/routes/instrumentation.ts please lmk if you need me to change

Reply:
I think the `for await` is not needed, you can change to `for`